src/main/java/com/headtrixz/ui/Home.java

```diff
@@ -20,4 +26,18 @@ public class Home {
     public void playTicTacToe() throws Exception {
         UIManager.switchScreen("game-mode");
     }
+
+    public void canPlay(Boolean bool){
+        playTicTacToeButton.setDisable(!bool);
+    }
+
+    public void connect() {
+        UIManager.setSetting("username", username.getText());
+        UIManager.setSetting("address", address.getText());
+        UIManager.setSetting("port", port.getText());
+
+        canPlay(true);
+
+        System.out.println("hoi");
```

Comment:
Laten we de stdout gebruiken voor daadwerkelijk debug/info messages, niet voor "hoi". 
Kan je deze print statement nog weghalen?